public/chromeos.html

```diff
@@ -0,0 +1,282 @@
+              />
+            </svg>
+          </div>
+        </section>
+        <section class="step" id="region-2">
+          <div class="step-half step-text">
+            <h2 class="step-title">2</h2>
+            <p>Select a region of the screen by clicking and dragging.</p>
```

Comment:
Can we make this text consistent with the other pages on the site (i.e. the pages for Windows and Mac)?

```suggestion
            <p>Click and drag to select the area to capture.</p>
```